SpatialGDK/Source/SpatialGDK/Public/EngineClasses/SpatialActorChannel.h

```diff
@@ -155,39 +155,27 @@ class SPATIALGDK_API USpatialActorChannel : public UActorChannel
 		return NetDriver->StaticComponentView->HasAuthority(EntityId, SpatialConstants::GetClientAuthorityComponent(GetDefault<USpatialGDKSettings>()->UseRPCRingBuffer()));
 	}
 
-	// Indicates whether this client worker has "ownership" (authority over Client endpoint) over the entity corresponding to this channel.
-	FORCEINLINE bool IsAuthoritativeClient() const
+	inline void OnClientAuthorityChange(Worker_AuthorityChangeOp Op)
```

Comment:
Realise it's inline, but just incase the compiler doesn't take the hint, make it a const&

Reply:
done